test-keeper.yml

```diff
@@ -0,0 +1,8 @@
+skip_validation_for: 
+  - 'Makefile'
+  - '.make/*'
+  - 'regex{{Gopkg.*}}'
+  - 'regex{{glide.*}}'
+  - '.dockerignore'
+  - 'Dockerfile.*'
+  - 'docker-compose.*'
```

Comment:
@xcoulon we support most of it out of the box https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/assets/config/test-keeper.yaml#L35 - can you contribute more if you find something missing?

Reply:
Oh, thanks for the link @bartoszmajsak! I was trying to make `Ike` happy until I realized that this PR also changes code in a `.go` file. However, let me see if the default settings are missing something.

Reply:
@bartoszmajsak here you are: https://github.com/arquillian/ike-prow-plugins/pull/112

Reply:
You are awesome! Thanks a lot @xcoulon 

![happy jonah hill gif-source](https://user-images.githubusercontent.com/719616/38725562-044c7f86-3f08-11e8-9ef2-8a5db8d43feb.gif)

Reply:
Keep the file here as is, I will make a clean-up PR when we apply your changes.